pinning.go

```diff
@@ -656,8 +672,8 @@ func (s *Server) handleAddPin(e echo.Context, u *User) error {
 
 	if s.CM.contentAddingDisabled || u.StorageDisabled {
 		return &util.HttpError{
-			Code:    http.StatusBadRequest,
-			Message: util.ERR_CONTENT_ADDING_DISABLED,
+			Code:   http.StatusBadRequest,
+			Reason: util.ERR_CONTENT_ADDING_DISABLED,
```

Comment:
I'm not sure if we need to change the `Message` field. We can add both `Message` and `Reason`.

Reply:
Actually, `Reason` is better for the internal error code in `util.XXXXXX_XXXX`. The `Message` field has been replaced with the `details` field - with has more message context. I just updated the error to add details.